here-naksha-app-service/src/test/java/com/here/naksha/app/service/NakshaAppTest.java

```diff
@@ -531,6 +531,123 @@ void tc0162_testUpdateEventHandlerWithMismatchingId() throws Exception {
     assertEquals(streamId, getHeader(response, HDR_STREAM_ID));
   }
 
+  @Test
+  @Order(3)
+  void tc0200_testCreateSpace() throws Exception {
+    // Test API : POST /hub/storages
+    // 1. Load test data
+    final String spaceJson = loadFileOrFail("TC0200_createSpace/create_space.json");
+    final String expectedBodyPart = loadFileOrFail("TC0200_createSpace/response.json");
+    final String streamId = UUID.randomUUID().toString();
+
+    // 2. Perform REST API call
+    final HttpRequest request = HttpRequest.newBuilder(stdHttpRequest, (k, v) -> true)
+        .uri(new URI(NAKSHA_HTTP_URI + "hub/spaces"))
+        .POST(HttpRequest.BodyPublishers.ofString(spaceJson))
+        .header(HDR_STREAM_ID, streamId)
+        .build();
+    final HttpResponse<String> response = httpClient.send(request, HttpResponse.BodyHandlers.ofString());
+
+    // 3. Perform assertions
+    assertEquals(200, response.statusCode(), "ResCode mismatch");
+    JSONAssert.assertEquals(
+        "Expecting new space in response", expectedBodyPart, response.body(), JSONCompareMode.LENIENT);
+    assertEquals(streamId, getHeader(response, HDR_STREAM_ID), "StreamId mismatch");
+  }
+
+  @Test
+  @Order(4)
+  void tc0201_testCreateDuplicateSpace() throws Exception {
+    // Test API : POST /hub/storages
+    // 1. Load test data
+    final String duplicatedSpace = loadFileOrFail("TC0201_createDupSpace/create_space.json");
```

Comment:
since the request body of TC0200 and TC0201 are the same (not just by coincident, by the design of the test itself "create the same space again"), it is possible to use the request body of the file from TC0200 here, and we can remove this from TC0201, so that we keep the code size stored within this git repo low.

Reply:
Sure, I can do that :) But I also don't think we should care that much about the repo size - as long as we're talking about number of small text files (I am against duplication & storing big data blobs but this is small json ;) ).

Reply:
No issue, it's an optional optimization :D